README.md

```diff
@@ -133,6 +133,23 @@ Feel free to mail me for any doubts/query
 ## :handshake: Contribution
 Feel free to **file a new issue** with a respective title and description on the the [Face-Mask-Detection](https://github.com/chandrikadeb7/Face-Mask-Detection/issues) repository. If you already found a solution to your problem, **I would love to review your pull request**! 
 
+## Awards
+Awarded Runners Up position in Amdocs Innovation India ICE Project Fair
+
+![](Readme_images/nn.jpeg)
+## 👏Appreciation
+Selected in [Devscript Winter Of Code])(https://devscript.tech/woc/)
+
+![](Readme_images/Devscript.jpeg)
+
+Selected in [Script Winter Of Code](https://swoc.tech/project.html)
+
+![](Readme_images/winter.jpeg)
+[Student Code-in](https://scodein.tech/)
```

Comment:
![Student Code-In](https://user-images.githubusercontent.com/29686102/103437821-26634000-4c52-11eb-8a00-c3999f2e43e5.jpeg)

Please insert this for Student Code-In and also write Selected in